models/perceptron.py

```diff
@@ -18,8 +18,7 @@ def __init__(self, input_size, output_size):
         self.b = sharedX(value=np.zeros(output_size), name='b', borrow=True)
 
     def initialize(self, weights_initializer=None):
```

Comment:
It would be simpler if we just change the default value for parameter `weights_initializer`
`def initialize(self, weights_initializer=WeightsInitializer().uniform):`